{"uid":"e8477a82a9412ba6","name":"test_deploy_active","fullName":"tests.integration.test_password_rotation#test_deploy_active","historyId":"7731bff68b9acca2f27c79bc5db0a391","time":{"start":1718156423387,"stop":1718156437682,"duration":14295},"description":"Build the charm and deploy it.","descriptionHtml":"<p>Build the charm and deploy it.</p>\n","status":"broken","statusMessage":"juju.errors.JujuAppError: App in error: postgresql-k8s","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: Optional[TResult] = func()\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 241, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 87, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 63, in thread_exception_runtest_hook\n    yield\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 90, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 65, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 850, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 833, in _runtest_for\n    yield\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 878, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 183, in pytest_runtest_call\n    raise e\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 173, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1632, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 162, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/tests/integration/test_password_rotation.py\", line 46, in test_deploy_active\n    await ops_test.model.wait_for_idle(apps=[APP_NAME], status=\"active\", timeout=1000)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 2963, in wait_for_idle\n    _raise_for_status(errors, \"error\")\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 2869, in _raise_for_status\n    raise error_type(\"{}{} in {}: {}\".format(\njuju.errors.JujuAppError: App in error: postgresql-k8s","flaky":false,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"event_loop","time":{"start":1718156423013,"stop":1718156423013,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1718156423013,"stop":1718156423013,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1718156423014,"stop":1718156423386,"duration":372},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1718156423386,"stop":1718156423386,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"Build the charm and deploy it.","status":"broken","statusMessage":"juju.errors.JujuAppError: App in error: postgresql-k8s","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: Optional[TResult] = func()\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 241, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 87, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 63, in thread_exception_runtest_hook\n    yield\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 90, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 65, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 850, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 833, in _runtest_for\n    yield\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 878, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 183, in pytest_runtest_call\n    raise e\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 173, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1632, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 162, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/tests/integration/test_password_rotation.py\", line 46, in test_deploy_active\n    await ops_test.model.wait_for_idle(apps=[APP_NAME], status=\"active\", timeout=1000)\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 2963, in wait_for_idle\n    _raise_for_status(errors, \"error\")\n  File \"/home/runner/work/test-runners-2-github-arm64-postgresql-k8s-operator/test-runners-2-github-arm64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 2869, in _raise_for_status\n    raise error_type(\"{}{} in {}: {}\".format(\njuju.errors.JujuAppError: App in error: postgresql-k8s","steps":[],"attachments":[{"uid":"fec7ee53f366e44a","name":"log","source":"fec7ee53f366e44a.txt","type":"text/plain","size":388}],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"abort_on_fail::0","time":{"start":1718156437715,"stop":1718156437715,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test::finalizer","time":{"start":1718156437731,"stop":1718156438126,"duration":395},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1718156438127,"stop":1718156438127,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1718156438127,"stop":1718156438127,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1718156438127,"stop":1718156438127,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"skip_if_deployed"},{"name":"tag","value":"abort_on_fail"},{"name":"tag","value":"group(1)"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration"},{"name":"suite","value":"test_password_rotation"},{"name":"host","value":"runner"},{"name":"thread","value":"13429-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.test_password_rotation"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"a756ac16af7bfbde","status":"unknown","time":{}}],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":1,"skipped":0,"passed":1,"unknown":3,"total":5},"items":[{"uid":"e45c6634d06997e1","reportUrl":"../75//#testresult/e45c6634d06997e1","status":"unknown","time":{}},{"uid":"f87bea1aac42f6e0","reportUrl":"../74//#testresult/f87bea1aac42f6e0","status":"unknown","time":{}},{"uid":"9c8eb336cb42400c","reportUrl":"../73//#testresult/9c8eb336cb42400c","status":"unknown","time":{}},{"uid":"3a7b80c6a6e8dcfd","reportUrl":"../72//#testresult/3a7b80c6a6e8dcfd","status":"passed","time":{"start":1717813473276,"stop":1717813804314,"duration":331038}}]},"tags":["skip_if_deployed","asyncio","abort_on_fail","group(1)"]},"source":"e8477a82a9412ba6.json","parameterValues":[]}